test/unit/profiles.saga.test.tsx

```diff
@@ -64,22 +77,21 @@ describe('fetchProfile behavior', () => {
     } as any)
   })
 
- it.skip('completes once for more than one request of same user',
-    () => {
-      return expectSaga(profileSaga)
-        .put(profileSuccess('passport' as any))
-        .not.put(profileSuccess('passport' as any))
-        .dispatch(profileRequest('user|1', future()))
-        .dispatch(profileRequest('user|1', future()))
-        .dispatch(profileRequest('user|1', future()))
-        .provide([
-          [select(getRealmAdapter), {}],
-          [call(profileServerRequest, 'user|1'), delayedProfile],
-          [select(getCurrentUserId), 'myid'],
-          [call(ensureAvatarCompatibilityFormat, profile), 'passport']
-        ])
-        .run()
-    })
+  it.skip('completes once for more than one request of same user', () => {
```

Comment:
Why is this test skipped?

Reply:
I'm not sure. They were like that from before. Maybe @pbosio can shed some light for us since from what I see, he was the one who committed the last time.

Reply:
after we migrated kernel to this repository, we find out that lots of test weren't running and, after making them run, lots of test were failing, so some test were fixed, others coded again and probably a couple skipped.
regarding this test in particular, I think the idea back then was to refactor the profile sagas in the short term, so we just skipped that test instead of fixing it.